alerting_profiles.tf

```diff
@@ -0,0 +1,36 @@
+resource "dynatrace_alerting" "di-alerting-profile" {
+    name            = "di-alerting-profile"
+    management_zone = ""
+    rules {
+        rule {
+            include_mode     = "NONE"
+            delay_in_minutes = 0
+            severity_level   = "AVAILABILITY"
+        }
+        rule {
+            include_mode     = "NONE"
+            delay_in_minutes = 0
+            severity_level   = "CUSTOM_ALERT"
+        }
+        rule {
+            include_mode     = "NONE"
+            delay_in_minutes = 0
+            severity_level   = "ERRORS"
+        }
+        rule {
+            include_mode     = "NONE"
+            delay_in_minutes = 0
+            severity_level   = "MONITORING_UNAVAILABLE"
+        }
+        rule {
+            include_mode     = "NONE"
+            delay_in_minutes = 0
```

Comment:
current settings 30 mins.

Reply:
Dynatrace default for resource contention and response time degradation is 30mins, although I'm not entirely sure why you'd want to be alerted 30mins after there has been a large spike in CPU or Memory or when your service has reduced performance and is operating slowly.
All this change means is that, when there is an issue, there won't be a delay in sending an alert into Slack. Alerts themselves are still controlled by the 3-in-5 rule, meaning three 1 minute periods in a 5 minute window will need to be above the threshold before firing an alert.
I'm not sure what the contractual agreement is with regards Live Service response times, but at the Home Office it was 30 minutes for a P1 incident; meaning in this circumstance you'd be responding outside of your contractual agreement to respond to an alert.